rollup/internal/controller/relayer/l2_relayer.go

```diff
@@ -553,6 +554,9 @@ type batchStatusResponse struct {
 }
 
 func (r *Layer2Relayer) getBatchStatusByIndex(batchIndex uint64) (bool, error) {
+	if r.chainMonitorClient == nil {
+		return true, nil
+	}
```

Comment:
how about move this check logic out of here , to the call place? L437?

Reply:
changed at https://github.com/scroll-tech/scroll/pull/942/commits/57b776070dc2b37528fdb6a950bca47d689b3280.